client/src/components/App/style.scss

```diff
@@ -1,8 +1,9 @@
+@import '../../styles/variables';
 .App {
-    background-color: blue;
-    color: aqua;
-    &__header {
-        font-size: 2rem;
-        color: aqua;
-    }
-}
\ No newline at end of file
+  background-color: $color-primary;
+  color: $text-color-secondary;
+  &__header {
+    font-size: 2rem;
+    color: $text-color-secondary;
+  }
+}
```

Comment:
make sure you have the same code formatting rules as the rest of the team 😄

Reply:
Could you tell me how it can be done? I mean, how can I check that?

Reply:
Ask on discord about it